manual/query_builder/insert/README.md

```diff
@@ -87,4 +87,20 @@ insertInto("user").json(bindMarker()).usingTimestamp(bindMarker())
 
 If you call the method multiple times, the last value will be used.
 
-[QueryBuilder]: http://docs.datastax.com/en/drivers/java/4.0/com/datastax/oss/driver/api/query-builder/QueryBuilder.html
+### Time To Live (TTL)
+
+Similar to Timestamps, you can specify data expiration TTLs with a literal value:
+
+```java
+insertInto("user").json(bindMarker()).usingTtl(500)
+// INSERT INTO user JSON ? USING TTL 500
```

Comment:
maybe our example should not use JSON to not complicate that? Only simple bindMarker for field and TTL to focus the attention of the reader on the TTL part of the example?